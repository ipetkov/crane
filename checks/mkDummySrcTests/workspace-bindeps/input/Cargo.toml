[package]
name = "workspace-bindeps"
version = "0.1.0"

[dependencies]
foo = { workspace = true }

[workspace.dependencies]
foo = { path = "./foo", target = "wasm32-unknown-unknown", artifact = "cdylib" }

