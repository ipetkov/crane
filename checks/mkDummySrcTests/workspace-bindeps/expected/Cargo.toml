[dependencies.foo]
workspace = true

[package]
build = "cranespecific-dummy.rs"
name = "workspace-bindeps"
version = "0.1.0"

[workspace.dependencies.foo]
artifact = "cdylib"
path = "./foo"
target = "wasm32-unknown-unknown"
